upgrade/sql/1.7.8.0.sql

```diff
@@ -199,3 +199,5 @@ CREATE TABLE IF NOT EXISTS `PREFIX_feature_flag` (
 INSERT INTO `PREFIX_feature_flag` (`name`, `state`, `label_wording`, `label_domain`, `description_wording`, `description_domain`)
 VALUES
 	('product_page_v2', 0, 'Experimental product page', 'Admin.Advparameters.Feature', 'This page benefits from increased performance and includes new features such as a new combination management system. Please note this is a work in progress and some features are not available yet.', 'Admin.Advparameters.Help');
+
+INSERT IGNORE INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES ('PS_LOGS_EMAIL_RECEIVERS', (SELECT `value` FROM `PREFIX_configuration` WHERE `name` = 'PS_SHOP_EMAIL'), NOW(), NOW());
```

Comment:
It should be in the 1.7.8.3.sql script too
Otherwise, upgrading to 1.7.8.3 from 1.7.8.2 will have no effect